[2025-08-15T22:28:34.963+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-08-15T22:28:35.007+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dbt_athena_pipeline.wait_orders_pipeline __airflow_temporary_run_2025-08-15T22:28:33.215729+00:00__ [queued]>
[2025-08-15T22:28:35.018+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dbt_athena_pipeline.wait_orders_pipeline __airflow_temporary_run_2025-08-15T22:28:33.215729+00:00__ [queued]>
[2025-08-15T22:28:35.019+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 3
[2025-08-15T22:28:35.037+0000] {taskinstance.py:2330} INFO - Executing <Task(ExternalTaskSensor): wait_orders_pipeline> on 2025-08-15 00:00:00+00:00
[2025-08-15T22:28:35.043+0000] {standard_task_runner.py:64} INFO - Started process 306 to run task
[2025-08-15T22:28:35.050+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dbt_athena_pipeline', 'wait_orders_pipeline', '__***_temporary_run_2025-08-15T22:28:33.215729+00:00__', '--job-id', '77', '--raw', '--subdir', 'DAGS_FOLDER/dbt_athena_pipeline.py', '--cfg-path', '/tmp/tmp2s8xsylf']
[2025-08-15T22:28:35.052+0000] {standard_task_runner.py:91} INFO - Job 77: Subtask wait_orders_pipeline
[2025-08-15T22:28:35.154+0000] {task_command.py:426} INFO - Running <TaskInstance: dbt_athena_pipeline.wait_orders_pipeline __airflow_temporary_run_2025-08-15T22:28:33.215729+00:00__ [running]> on host e1203d83167e
[2025-08-15T22:28:35.310+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dbt_athena_pipeline' AIRFLOW_CTX_TASK_ID='wait_orders_pipeline' AIRFLOW_CTX_EXECUTION_DATE='2025-08-15T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='__***_temporary_run_2025-08-15T22:28:33.215729+00:00__'
[2025-08-15T22:28:35.311+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-08-15T22:28:35.327+0000] {baseoperator.py:400} WARNING - ExternalTaskSensor.execute cannot be called outside TaskInstance!
[2025-08-15T22:28:35.346+0000] {external_task.py:274} INFO - Poking for DAG 'orders_pipeline' on 2025-08-15T00:00:00+00:00 ... 
[2025-08-15T22:28:35.348+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.11/site-packages/***/utils/session.py:79: DeprecationWarning: This method is deprecated and will be removed in future.
  return func(*args, session=session, **kwargs)

[2025-08-15T22:28:35.371+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-08-15T22:28:35.406+0000] {taskinstance.py:2865} INFO - Rescheduling task, marking task as UP_FOR_RESCHEDULE
[2025-08-15T22:28:35.425+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2025-08-15T22:28:35.470+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-08-15T22:28:35.473+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
